client/src/components/pages/not-found/not-found.tsx

```diff
@@ -1,12 +1,35 @@
+import "./not-found.css"
+import { FormattedMessage } from "react-intl";
+import beerTap from "@/assets/images/beer-tap.png";
 import BackToHome from "@/components/ui/back-to-home/back-to-home"
 
+
+
 const NotFound = () => {
   return(
-    <main className="not-found">
-      <h1>404</h1>
-      <p>Page Not Found</p>
+
+    <>
```

Comment:
In general, more specific html tags are preferred for accessibility.

```suggestion
    <main>
```